test/integration/logic/TypeResolverTest.java

```diff
@@ -670,14 +666,14 @@ public void simple_always_infers_its_supers() {
         }};
 
         assertEquals(expected, getHintMap(exhaustiveConjunction));
-        assertEquals(expected, getHintMap(simpleConjunction));
     }
 
     // When a hint label exists, it can "skip" a generation, meaning a hint and the hint's descendent is possible, yet
     // none of the hint's direct children are possible.
     // We show this below on the hint labels of $t
     // We also show on $a that hints can be isolated form each other completely hierarchy-wise.
     @Test
+    //TODO: ignored as the gap fpesn't appea. Unclear if this is a resolver or traversal bug.
```

Comment:
stray TODO

Reply:
Not a stray this time. This is the query that I discoed led to a "gap" in the resolved types. Except the gap ins't appearing! I don't think this is a pressing issue current, so I have left it as a TODO.

Reply:
Oh, why is the test enabled then?

Reply:
It isn't, there's "@ignore" right below that line.